pages/common/while.md

```diff
@@ -0,0 +1,11 @@
+# while
+
+> Simple shell loop
+
+- Read stdin and perform an action on every line.
+
+`while read line; do {{echo "$line"}}; done`
```

Comment:
Since this is a concrete example, I don't think echo "$line" should use placeholder syntax. But this is an interesting use case indeed -- should we distinguish concrete examples like we distinguish placeholders? Perhaps highlight them to differentiate from the command's own structure, but in a different way than generic tokens? Calling @igorshubovych @ostera @rprieto @felixonmars @rubenvereecken